package.json

```diff
@@ -34,12 +34,13 @@
     "yalc:publish": "lerna run yalc:publish"
   },
   "volta": {
-    "node": "16.19.0",
```

Comment:
Actually, thinking about this, can we do this in a separate PR?

Reply:
Unfortunately, this was already merged with #9187 :( 
(this PR still needs a rebase)

I agree it would have been better to split this into a separate PR.

Reply:
ah, right, well whatever, not a big deal ;)